bundles/framework/timeseries/WMSAnimator.js

```diff
@@ -132,6 +132,8 @@ Oskari.clazz.define('Oskari.mapframework.bundle.timeseries.WMSAnimator',
                         me._resolveWait();
                     });
                 }
+                const layerParams = this._layer.getParams();
+                layerParam.time = newTime;
```

Comment:
There's an `s` missing from variable name?
```suggestion
                layerParams.time = newTime;
```